packages/ui/src/molecules/QuantitySelector/stories/QuantitySelector.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import type { Meta, Story } from '@storybook/react'
+import React, { useState, useMemo } from 'react'
+
+import type {
+  QuantitySelectorProps,
+  QuantitySelectorButtonProps,
+  QuantitySelectorInputProps,
+} from '..'
+import {
+  QuantitySelector,
+  QuantitySelectorButton,
+  QuantitySelectorInput,
+} from '..'
+import { PlusIcon, MinusIcon } from './assets/Icons'
+import mdx from './QuantitySelector.mdx'
+
+const MAX_QUANTITY = 10
+const MIN_QUANTITY = 1
+
+type QuantitySelectorCompoundProps = QuantitySelectorProps &
+  QuantitySelectorButtonProps &
+  QuantitySelectorInputProps
+const QuantitySelectorStylelessTemplate: Story<QuantitySelectorCompoundProps> = (
+  args
+) => {
+  const [quantity, setQuantity] = useState(MIN_QUANTITY)
+
+  function isLeftDisabled() {
+    return quantity === MIN_QUANTITY
+  }
+
+  function isRightDisabled() {
+    return quantity === MAX_QUANTITY
+  }
+
+  return (
+    <QuantitySelector
+      {...args}
+      name="quantity-selector"
+      currentValue={quantity}
+      onClick={(v) => {
+        if (v.currentTarget.name === 'quantity-selector-increase-button') {
+          setQuantity((currentQuantity) =>
+            Math.min(currentQuantity + 1, MAX_QUANTITY)
+          )
+        } else if (
+          v.currentTarget.name === 'quantity-selector-decrease-button'
+        ) {
+          setQuantity((currentQuantity) =>
+            Math.max(currentQuantity - 1, MIN_QUANTITY)
+          )
+        }
+      }}
+    >
+      <QuantitySelectorButton
+        name="quantity-selector-decrease-button"
+        icon={<MinusIcon color="#fff" />}
+        disabled={isLeftDisabled()}
+      />
+      <QuantitySelectorInput name="'quantity-selector-input'" readOnly />
```

Comment:
You are using this input without a `<label>`, which is not recommended.

> Form <input> elements must have labels 
Find out more about it: https://dequeuniversity.com/rules/axe/4.3/label?application=axeAPI

We are not using any label to identify this input on `QuantitySelectorInput`, Maybe we could add a default `aria-label=" Quantity Selector"` or receive it from it's as a prop. What do you think?

![image](https://user-images.githubusercontent.com/3356699/144881471-cf129ef3-cb51-47fc-9f04-22f9ad321945.png)

Reply:
Good idea. Thanks for the heads up! Lemme know what you think about https://github.com/vtex/faststore/pull/1035/commits/dfaaee45a47e61f6968277bc4f215e5c6a78bd9c.